tests/test_session.py

```diff
@@ -677,6 +677,50 @@ def test_app_utilities_new_and_old(mixing_elbow_settings_session):
 
 
 @pytest.mark.standalone
+def test_new_launch_fluent_api():
+    import ansys.fluent.core as pyfluent
+
+    solver = pyfluent.Solver.from_install()
+    assert solver.health_check.check_health() == "SERVING"
+
+    ip = solver.connection_properties.ip
+    password = solver.connection_properties.password
+    port = solver.connection_properties.port
+
+    solver_connected = pyfluent.Solver.from_connection(
+        ip=ip, password=password, port=port
+    )
+    assert solver_connected.health_check.check_health() == "SERVING"
+
+    solver.exit()
+    solver_connected.exit()
+
+
+def test_new_launch_fluent_api_from_container():
+    import ansys.fluent.core as pyfluent
+    from ansys.fluent.core.utils.networking import get_free_port
+
+    port_1 = get_free_port()
+    port_2 = get_free_port()
+    container_dict = {"ports": {f"{port_1}": port_1, f"{port_2}": port_2}}
+    solver = pyfluent.Solver.from_container(container_dict=container_dict)
+    assert solver.health_check.check_health() == "SERVING"
+    solver.exit()
+
+
+def test_new_launch_fluent_api_from_connection():
+    import ansys.fluent.core as pyfluent
+
+    solver = pyfluent.Solver.from_container()
+    assert solver.health_check.check_health() == "SERVING"
+    ip = solver._fluent_connection.connection_properties.ip
+    port = solver._fluent_connection.connection_properties.port
+    password = solver._fluent_connection.connection_properties.password
```

Comment:
If we want the connection properties to be exposed to users like this should we expose them from the high level (session) like we planned to do for "register_finalizer_cb". Please share your views @mkundu1 and @seanpearsonuk. @hpohekar, this might not be part of this PR, if it is required I can take care of that in my PR which I have for the other case.

Reply:
@prmukherj  `solver.connection_properties` exists already. We are using this now.

Reply:
great. 👍